parsers/AR.py

```diff
@@ -533,7 +539,7 @@ def fetch_price(country_code='AR', session = None):
         rprice = lprice.split('[')[0]
         price = float(rprice.replace(',','.'))
 
-    except AttributeError, ValueError:
+    except AttributeError as ValueError:
```

Comment:
This still catches both errors I assume?

Reply:
Good catch.  I did this wrong.  Fixing now.

This is exactly why Python 3 changed the syntax.  The old way was too ambiguous.